src/SDKs/_metadata/batch_data-plane.txt

```diff
@@ -1,11 +1,11 @@
-2018-04-24 22:42:09 UTC
+2018-08-21 16:42:27 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: Azure
-Branch:      master
-Commit:      e6d46513662f00ab4d0d560ab710be4a849609c3
+GitHub user: matthchr
```

Comment:
Is the REST spec merged? We cannot accept this PR until the spec is merged to Azure master

Reply:
It's already merged I just generated from a private branch from before and haven't updated this file.

I can regenerate it if you want.

Reply:
That would be much appreciated. If you can pull down latest changes and run `msbuild build.proj` before regenerating that'd be even better